vitest.examples.config.ts

```diff
@@ -0,0 +1,24 @@
+import { defineConfig } from "vitest/config";
+import tsConfigPaths from "vite-tsconfig-paths";
+import packageJson from "./package.json";
+
+export default defineConfig({
+  test: {
+    globals: true,
+    passWithNoTests: true,
+    testTimeout: 120 * 1000,
+    printConsoleTrace: true,
+    setupFiles: ["./tests/setup.examples.ts"],
+    deps: {
+      interopDefault: false,
+    },
+  },
+  define: {
+    __LIBRARY_VERSION: JSON.stringify(packageJson.version),
+  },
+  plugins: [
+    tsConfigPaths({
+      projects: ["tsconfig.json"],
+    }),
```

Comment:
This seems wrong - you should use the `tsconfig.examples.json`

Reply:
I seems that the typescript config file for "/tests/examples/examples.test.ts" must be "tsconfig.json".